engine/frontend/src/component/PackageCatalog.js

```diff
@@ -45,12 +45,13 @@ const PackageCatalog = ({kurtosisPackages: defaultPackages}) => {
         }
         const filteredPackages = defaultPackages.filter(pack => {
                 if ("name" in pack) {
-                    return pack.name.includes(value)
+                    // lowercase everything so that it works for both cases
+                    const trimmedValue = value.trim()
```

Comment:
@Peeeekay do we need to check for undefined and null pointer?

Reply:
Same comment other places

Reply:
The default value is `""`; the initial state for all the inputs is an empty string.. so it shouldn't be an issue